data/z_text.data.s

```diff
@@ -7,11 +7,11 @@
 
 .section .data
 
+# temporary name, rename to something more appropriate when decompiled
```

Comment:
I would make it clear you're talking about the file name here, just to avoid any confusion in the future.
```suggestion
# temporary file name, rename to something more appropriate when decompiled
```